ArduCopter/GCS_Mavlink.cpp

```diff
@@ -879,7 +879,7 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
             float north = packet.param1;
             float east = packet.param2;
             float up = packet.param3 + (copter.inertial_nav.get_position().z / 100.);
-            copter.planck_interface.request_move_target(Vector3f(north,east,up));
+            copter.planck_interface.request_move_target(Vector3f(north,east,up),false,copter.pos_control->get_max_speed_up(),copter.pos_control->get_max_speed_down());
```

Comment:
are the `get_max_speed_...()` methods correct? Do they return the value of `WPNAV_SPEED_UP/DN`?

Reply:
Yes, they get the get_max_speed_...()  functions get the limits from the position controller, which are set to the WPNAV_SPEED_UP/DOWN values when in pos_vel on angle control. These functions are used in the RTB command messages as well